content/views.py

```diff
@@ -0,0 +1,4 @@
+"""Purpose of this file
+
+This file describes the frontend views.
```

Comment:
Same comment as above.
All frontend views have already been outsourced to the frontend app